backend/app/src/proto/file.proto

```diff
@@ -0,0 +1,11 @@
+package userpackage;
+syntax = "proto3";
+
+mssage GameInfo {
```

Comment:
Rewrite mssage to message to make the server works

Reply:
Oops, sorry about that typo just before committing, fixed!